dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -736,13 +736,41 @@ def test_profile_merge_edge_case(self):
         self.assertEqual(profiler.min, 2.0)
         self.assertEqual(profiler.max, 5.0)
 
+    def test_custom_bin_count_merge(self):
+
+        options = FloatOptions()
+        options.histogram_and_quantiles.method = None
+        options.histogram_and_quantiles.hist_bin_count = 10
+
+        data = [2.0, 'not a float', 6.0, 'not a float']
+        df = pd.Series(data).apply(str)
+        profiler1 = FloatColumn("Float", options)
+        profiler1.update(df)
+
+        data2 = [10.0, 'not a float', 15.0, 'not a float']
+        df2 = pd.Series(data2).apply(str)
+        profiler2 = FloatColumn("Float", options)
+        profiler2.update(df2)
+
+        # no warning should occur
+        with warnings.catch_warnings(record=True) as w:
+            profiler1 + profiler2
+        self.assertListEqual([], w)
```

Comment:
can we add a test to assert `profiler2.user_set_histogram_bin = 10`? and also for profiler1?

Reply:
^ I agree

Reply:
the intent of this function is to test merging when custom bin is set, not individual options.
However, i did add: `self.assertEqual(10, merge_profile.user_set_histogram_bin)`

Additionally, the test to check if options propagate and effect final results occurs in: `test_histogram_option_integration` in the area commented: `# test histogram bin count set`